[2022-03-17 03:20:54,937] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-04-02T06:00:00+00:00 [queued]>
[2022-03-17 03:20:54,974] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-04-02T06:00:00+00:00 [queued]>
[2022-03-17 03:20:54,975] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:20:54,977] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:20:54,979] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:20:55,004] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-04-02T06:00:00+00:00
[2022-03-17 03:20:55,010] {standard_task_runner.py:52} INFO - Started process 909 to run task
[2022-03-17 03:20:55,018] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-04-02T06:00:00+00:00', '--job-id', '18', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmpxkhz15t5', '--error-file', '/tmp/tmp_ieo5u85']
[2022-03-17 03:20:55,024] {standard_task_runner.py:77} INFO - Job 18: Subtask ingest
[2022-03-17 03:20:55,130] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-04-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:20:55,249] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T06:00:00+00:00
[2022-03-17 03:20:55,253] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:20:55,256] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:20:55,270] {bash.py:169} INFO - Output:
[2022-03-17 03:20:55,334] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:20:55,336] {bash.py:173} INFO - dags
[2022-03-17 03:20:55,337] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:20:55,339] {bash.py:173} INFO - logs
[2022-03-17 03:20:55,341] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:20:55,343] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:20:55,345] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:20:55,346] {bash.py:173} INFO - output_2019-04.csv
[2022-03-17 03:20:55,348] {bash.py:173} INFO - output_2019-05.csv
[2022-03-17 03:20:55,349] {bash.py:173} INFO - output_2019-06.csv
[2022-03-17 03:20:55,351] {bash.py:173} INFO - plugins
[2022-03-17 03:20:55,352] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:20:55,353] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:20:55,354] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:20:55,399] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190402T060000, start_date=20220317T032054, end_date=20220317T032055
[2022-03-17 03:20:55,456] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:20:55,472] {local_task_job.py:146} INFO - Task exited with return code 0
